src/services/users.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const Users = require('../db/models/users');
+
+function normalizeUser({ id, name }) {
```

Comment:
normalizeUser function could be simplified using object destructuring: function normalizeUser(user) { return { id: user.id, name: user.name }; }